hooks/conan-center.py

```diff
@@ -1231,6 +1236,14 @@ def test(out):
             for lib in libs:
                 out.warn("Library '{}' links to system library '{}' but it is not in cpp_info.{}.".format(lib, missing_system_lib, attribute))
 
+    @run_test("KB-H077", output)
+    def test(out):
+        if not is_apple_os(conanfile):
+            return
+        not_relocatable_libs = _get_non_relocatable_shared_libs(conanfile)
+        if not_relocatable_libs:
+            out.error(f"install_name dir of these shared libs is not @rpath: {', '.join(not_relocatable_libs)}")
```

Comment:
```suggestion
            out.warn(f"install_name dir of these shared libs is not @rpath: {', '.join(not_relocatable_libs)}")
```
Please, keep as warning level, because we have a bad in master branch, it will break master in CI jenkins.

Reply:
What does it break in c3i for master branch?

Reply:
CCI Jenkins has an internal build named "ScheduledExportCheck", which runs everyday and execute `conan export`. But looking again, this PR should not affect it, as it a post package step.